x-pack/plugins/monitoring/public/components/elasticsearch/nodes/nodes.js

```diff
@@ -21,7 +21,9 @@ import {
 } from '@elastic/eui';
 import { i18n } from '@kbn/i18n';
 import { injectI18n } from '@kbn/i18n/react';
+import _ from 'lodash';
 
+const getSortHandler = (type) => (item) => _.get(item, `${type}.summary.lastVal`);
```

Comment:
I don't know why, but for this some reason, this isn't working for me.

```
const getSortHandler = (type) => (item) => _.get(item, [type, 'summary', 'lastVal']);
```

This does work for me and I don't know why. Can you confirm/deny?

Reply:
Bump @igoristic. It would be great if we could get this resolved and then this fix merged. Thanks!

Reply:
Update: I didn't closely read the time stamp here and left the impression that this is urgent when it's not. Apologies if it came across like I was pushing for this to be done. It will be fine whenever you get to it. Thanks!